src/interactions/stories/Interactions.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import { Meta } from "@storybook/react";
+import React, { FocusEvent, KeyboardEvent } from "react";
+
+import { Press } from "../Press";
+import { Focus } from "../Focus";
+import { Keyboard } from "../Keyboard";
+import { FocusWithin } from "../FocusWithin";
+import { FocusVisible } from "../FocusVisible";
+import { Hover } from "../Hover";
+
+export default {
+  title: "Component/Interactions",
+} as Meta;
+
+export const Default = () => {
```

Comment:
Should we also create a story for each component? (Focus, Hover, Focusable, Keyboard etc etc)

Reply:
May be later, now we are only using it for our purpose.

I thought we can make these stories proper when we are writing the proper docs.

Reply:
Okay!